lib/virt_autotest/utils.pm

```diff
@@ -1269,7 +1269,12 @@ sub get_guest_regcode {
     my $regcode_ltss = get_var("GUEST_SCC_REGCODE_LTSS", "");
     my $count = ($args{separator} eq '|' ? scalar(split("\\$args{separator}", $guest)) : scalar(split("$args{separator}", $guest)));
     $regcode = join("$args{separator}", (get_var("SCC_REGCODE", "")) x $count) if (!$regcode);
-    $regcode_ltss = join("$args{separator}", (get_var("SCC_REGCODE_LTSS_15", "")) x $count) if (!$regcode_ltss);
+    my @guest_parts = split($args{separator}, $guest);
```

Comment:
I think the pr is ok but this block,  if `GUEST_SCC_REGCODE_LTSS` is set(the most cases in x86 ltss tests), the following statements for `@regcode_ltss_parts` is not required at all, but this change add an `for` block for it and finally it is not used, it is excessive. If  `if (!$regcode_ltss)` is moved backward, we would save a little bit by skipping handling the useless `@regcode_ltss_parts`.

Reply:
That's the problem, will improve, thanks Julie.